src/core/scene/a-scene.js

```diff
@@ -294,9 +297,13 @@ class AScene extends AEntity {
         var xrMode = useAR ? 'immersive-ar' : 'immersive-vr';
         xrInit = this.sceneEl.systems.webxr.sessionConfiguration;
         return new Promise(function (resolve, reject) {
-          navigator.xr.requestSession(xrMode, xrInit).then(
+          var requestFunction = useOfferSession ? navigator.xr.offerSession.bind(navigator.xr) : navigator.xr.requestSession.bind(navigator.xr);
+          self.usedOfferSession |= useOfferSession;
```

Comment:
can just do assign here

Reply:
Since this same function is called for regular requestSession in response to a button click, we need to take some extra steps here.
This code is also not correct. I will update in the next diff.